src/Stryker.Core/Stryker.Core/MutationTest/MutationTestProcess.cs

```diff
@@ -200,12 +200,20 @@ public void Optimize(TestCoverageInfos coveredMutants)
                 {
                     continue;
                 }
-                mutant.CoveringTest = tests.Select(x => x.ToString()).ToList();
+                if (mutant.IsStaticValue)
+                {
+                    Logger.LogDebug($"Mutant {mutant.DisplayName} is related to a static value and we cannot have reliable coverage info. No optimization done.");
+                }
+                else
+                {
+                    mutant.CoveringTest = tests.Select(x => x.ToString()).ToList();
+                }
             }
 
             report.AppendJoin(',', nonTested.Select(x => x.Id));
             Logger.LogInformation(nonTested.Count == 0
                 ? "Congratulations, all mutants are covered by tests!"
+
```

Comment:
This can be removed?

Reply:
This is not new. What is your concern with this message?

Reply:
Sorry it was not about the message but the newline.